packages/frontapp/src/app/components/BookInfo/index.tsx

```diff
@@ -80,55 +95,197 @@ const OpenLink = styled.a`
   line-height: ${dimensions.xl};
   text-decoration: underline;
 `;
+
 const Description = styled.p`
   font-weight: 300;
   font-size: ${dimensions.base};
   line-height: ${dimensions.xl};
 `;
+
+const StyledButton = styled(Button)`
+  cursor: pointer;
+  margin-bottom: ${dimensions.xs_2};
+  padding: 0 2rem;
+  :hover {
+    background-color: ${colors.hover_color};
+    color: ${colors.bg_primary};
+  }
+`;
+
 interface IProps {
   src: string | null | undefined;
   title: string | undefined;
   description: string | undefined;
-  status: string | null | undefined;
+  status: string | undefined;
   author: string | undefined;
   category: string | string[] | undefined;
-  created_at: string | undefined;
+  identifier: string;
+  created_at: any;
 }
```

Comment:
A lot of props here. I guess, It will be better to replace it with one prop. And props interface rename to "IBookInfoProps"